code/datums/traits/_quirk.dm

```diff
@@ -12,6 +12,7 @@
 	var/antag_removal_text // Text will be given to the quirk holder if they get an antag that has it blacklisted.
 	var/mood_quirk = FALSE //if true, this quirk affects mood and is unavailable if moodlets are disabled
 	var/mob_trait //if applicable, apply and remove this mob trait
+	var/mob_extratrait //for the occasional quirk that adds more then one trait
```

Comment:
Could just be a list to avoid copy paste

Reply:
So just using mob_trait = list(the list) didn't work. I ended up just making it so the quirk has a datum that just adds the two traits. Bit blunt, but functional and doesn't mess with quirk.dm at all.